config/stack_custom.yaml

```diff
@@ -90,86 +90,86 @@ tau2:
 
 bacio:
   build: YES
-  version: v2.4.0
-  install_as: 2.4.0
+  version: v2.4.1
+  install_as: 2.4.1
   openmp: OFF
 
 sigio:
   build: YES
-  version: v2.3.0
-  install_as: 2.3.0
+  version: v2.3.2
+  install_as: 2.3.2
   openmp: OFF
 
 sfcio:
   build: YES
-  version: v1.4.0
-  install_as: 1.4.0
+  version: v1.4.1
+  install_as: 1.4.1
   openmp: OFF
 
 gfsio:
   build: YES
-  version: v1.4.0
-  install_as: 1.4.0
+  version: v1.4.1
+  install_as: 1.4.1
   openmp: OFF
 
 w3nco:
   build: YES
-  version: v2.4.0
-  install_as: 2.4.0
+  version: v2.4.1
+  install_as: 2.4.1
   openmp: OFF
 
 sp:
   build: YES
-  version: v2.3.0
-  install_as: 2.3.0
+  version: v2.3.1
+  install_as: 2.3.1
   openmp: OFF
```

Comment:
reason to keep this `OFF`?

Reply:
No

Is there a reason in general to disable OpenMP by default? I changed SP because I know that SP in particular makes heavy use of it to calculate

Reply:
No. There is no reason to turn it OFF.  It was the default in our NCEPLIBS, and I stuck with that in my iteration.
Just like I was building shared libs, but now we have turned them to static.